concordium-consensus/tests/consensus/ConcordiumTests/KonsensusV1/TransactionProcessingTest.hs

```diff
@@ -347,9 +349,81 @@ dummyTransaction =
             }
     payload = encodePayload $ Transfer dummyAccountAddress 10
 
--- | The block item for 'dummyTransaction'.
-dummyTransactionBI :: BlockItem
-dummyTransactionBI = normalTransaction dummyTransaction
+-- | A dummy update instruction.
+--
+-- TODO (drsk) The signature needs to be changed to a correct one.
```

Comment:
I haven't figured out so far how to add the right keys for the update instruction.